src/pyshark/config.py

```diff
@@ -4,10 +4,19 @@
 
 import pyshark
 
-CONFIG_PATH = os.path.join(os.path.dirname(pyshark.__file__), 'config.ini')
+
+fp_config_path = os.path.join(os.getcwd(), 'config.ini')  # get config from the current directory
+pyshark_config_path = os.path.join(os.path.dirname(pyshark.__file__), 'config.ini')
 
 
 def get_config():
+    if os.path.exists(fp_config_path):
+        CONFIG_PATH = fp_config_path
```

Comment:
Change this to lowercase sicne it's not a const anymore please.
Also if you could take the time to use `pathlib` instead of `os.path` that'd be great. I want to change it everywhere in the code.